rocAL/rocAL/include/augmentations/color_augmentations/node_blend.h

```diff
@@ -25,19 +25,16 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 
-class BlendNode : public Node
-{
+class BlendNode : public Node {
```

Comment:
These file changes only seem to contain formatting changes.
If we have many such similar files, can we issue that as a seperate PR?
Or revert the file changes if only a single file has such a case

Reply:
Since we planned to issue all these in a single PR we have added changes in all the augmentation node files

Reply:
Removed